src/components/general/Tabs/Tab.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import * as styles from "./styles.less";
+import * as classNames from "classnames";
+import * as PropTypes from "prop-types";
+
+type TabProps = {
+    isCurrent?: boolean,
+    title: string,
+    disabled?: boolean,
+    onTabClick: () => void,
+};
+
+const Tab = ({ isCurrent, title, disabled, onTabClick }: TabProps) => {
+    const [isHovering, setIsHovering] = React.useState(false);
+    const [isPressed, setIsPressed] = React.useState(false);
+
+    const tabClasses = classNames(styles.tab, {
+        [styles.current]: isCurrent,
+        [styles.hover]: isHovering && !disabled,
+        [styles.pressed]: isPressed && !disabled,
+        [styles.disabled]: disabled,
+    });
+    const titleClasses = classNames(styles.title);
+
+    return (
+        <button
```

Comment:
Should we consider using anchor-tags instead of buttons? That way developers could provide a url/href property to Tab and we would redirect to that url. This would enencourage the use of url-navigation and allow users to open tabs in new... tabs